MPF.Processors/Redumper.cs

```diff
@@ -184,16 +192,29 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, string basePath
                         info.CommonDiscInfo.Region = ProcessingTool.GetXGDRegion(xmid.Model.RegionIdentifier);
                     }
 
-                    if (HashTool.GetStandardHashes($"{basePath}.manufacturer", out _, out string? dmi1Crc, out _, out _))
-                        info.CommonDiscInfo!.CommentsSpecialFields![SiteCode.DMIHash] = dmi1Crc ?? string.Empty;
-                    if (HashTool.GetStandardHashes($"{basePath}.physical", out _, out string? pfi1Crc, out _, out _))
-                        info.CommonDiscInfo!.CommentsSpecialFields![SiteCode.PFIHash] = pfi1Crc ?? string.Empty;
+                    if (HashTool.GetStandardHashes($"{basePath}.dmi", out _, out string? dmi1Crc, out _, out _))
```

Comment:
Please use `HashTool.GetFileHash(path, HashType.CRC32)` to avoid unnecessary extra hashing work.